### PR TITLE
fix(notion): Fix Issue with Skipped Items within the Same Minute Interval

### LINKED ISSUE #4671: Notion updated trigger - skipping items
**Describe the bug:**  
The Notion API returns the data rounded to the nearest minutes, which means it can skip the rest of the items after the last fetched item if they are in the same minute.

This issue was found by @HamzaZagha

### PR DESCRIPTION
**Pull request by kishanprmr**
## What does this PR do?

Fixes #4671 

ANNOUNCEMENT=true

### CODE REVIEW

File: packages/pieces/community/notion/src/index.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const notion = createPiece({
 	description: 'The all-in-one workspace',
 	logoUrl: 'https://cdn.activepieces.com/pieces/notion.png',
 	categories: [PieceCategory.PRODUCTIVITY],
-	minimumSupportedRelease: '0.5.0',
+	minimumSupportedRelease: '0.26.2',
```
Comment (abuaboud) [inline packages/pieces/community/notion/src/index.ts]: Can you please revert this

File: packages/pieces/community/notion/src/lib/triggers/new-database-item.ts [hunk 2]
```diff
@@ -146,22 +156,36 @@ const getResponse = async (
...
+    const response = await notion.databases.query({
+      start_cursor: cursor,
+      database_id,
+      filter:
+        startDate == null
+          ? undefined
+          : {
+              timestamp: 'created_time',
+              created_time: {
+                after: startDate,
```
Comment (abuaboud) [inline packages/pieces/community/notion/src/lib/triggers/new-database-item.ts]: shouldn't we use the one with >= instead?

File: packages/pieces/community/notion/src/lib/triggers/updated-database-item.ts [hunk 3]
```diff
@@ -146,22 +157,36 @@ const getResponse = async (
...
+    const response = await notion.databases.query({
+      start_cursor: cursor,
+      database_id,
+      filter:
+        startDate == null
+          ? undefined
+          : {
+              timestamp: 'last_edited_time',
+              last_edited_time: {
+                after: startDate,
```
Comment (abuaboud) [inline packages/pieces/community/notion/src/lib/triggers/updated-database-item.ts]: same here

**Review by abuaboud — CHANGES_REQUESTED** [review]

File: packages/pieces/community/notion/src/index.ts (hunk 1 above)
Comment (kishanprmr) [reply packages/pieces/community/notion/src/index.ts]: fixed it.

**Review by abuaboud — APPROVED** [review]

**Pull request merged**